src/lib/support/Pool.h

```diff
@@ -138,7 +138,7 @@ class LambdaProxy
 
 struct HeapObjectListNode
 {
-    void Remove()
+    void Remove() const
```

Comment:
This is not conceptually const either (e.g. if it did not leave mNext/mPrev dangling, which arguably it should not, it could not be const...)